packages/react-avatar/src/Avatar.stories.tsx

```diff
@@ -170,15 +196,15 @@ export const ActiveAnimation = () => {
   const [size, nextSize, prevSize] = useValueSelectorState(examples.size, 96);
   const [activeDisplay, nextActiveDisplay, prevActiveDisplay] = useValueSelectorState(examples.activeDisplay, 'ring');
   const [display, nextDisplay, prevDisplay] = useValueSelectorState(['image', 'icon', 'label'], 'image');
-
+  const flexStyles = useFlexStyles();
   React.useEffect(() => {
     const id = setTimeout(() => setActive(true), 500);
     return () => clearTimeout(id);
   }, []);
 
   return (
-    <ThemeProvider>
-      <Stack>
+    <>
```

Comment:
minor nit: this fragment isn't necessary since it just has one div as a child
```suggestion
```

Reply:
Agree, removed it 👍